server/db/models/home_place.js

```diff
@@ -1,15 +1,89 @@
 const Sequelize = require('sequelize')
 const db = require('../db')
+const Place = require('./place')
+const Home = require('./home')
+const Location = require('./location')
+const {getDistanceFromGoogle} = require('../../services')
 
 const HomePlace = db.define('home_place', {
   id: {
     type: Sequelize.INTEGER,
     primaryKey: true,
     autoIncrement: true
   },
-  distance: {
+  distanceValue: {
     type: Sequelize.INTEGER
+  },
+  distanceText: {
+    type: Sequelize.STRING
+  },
+  walkingValue: {
+    type: Sequelize.INTEGER
+  },
+  walkingText: {
+    type: Sequelize.STRING
+  },
+  drivingValue: {
+    type: Sequelize.INTEGER
+  },
+  drivingText: {
+    type: Sequelize.STRING
+  },
+  bicyclingValue: {
+    type: Sequelize.INTEGER
+  },
+  bicyclingText: {
+    type: Sequelize.STRING
+  },
+  transitValue: {
+    type: Sequelize.INTEGER
+  },
+  transitText: {
+    type: Sequelize.STRING
+  }
+})
+
+HomePlace.afterCreate(async instance => {
+  console.log('Home Id:', instance.homeId)
+  console.log('Place Id:', instance.placeId)
```

Comment:
are this console logs of any value? If they were just for debugging lest's remove them